src/core/sort.cc

```diff
@@ -739,6 +740,7 @@ class SortContext {
                      : static_cast<uint8_t>(128 - xi[j]) >> 6;
         });
     }
+    RESTORE_CLANG_WARNING("-Wpadded")
```

Comment:
So no way to fix this besides disabling?

Reply:
the warning is in the auto-generated lambda code, so not much. Other than rewriting the lambda itself, but that doesn't seem possible here